src/components/autofill/autofill.component.js

```diff
@@ -7,37 +7,46 @@ import PreviewItem from "./preview-item/preview-item.component";
 
 import { SEARCH_RESULTS_PAGE } from "./../../root/root.constants";
 
+import { goCrewViewPage, goVesselViewPage, goBoardingViewPage } from './../../helpers/get-data'
+
 import "./autofill.css";
 
 class Autofill extends Component {
   render() {
     const { t, vessels, crew, boardings, searchQuery, searchWords } = this.props;
 
+    const crewItem = crew[0]
+    const vesselItem = vessels[0]
+    const boardingItem = boardings[0]
+
     return (
       <div className="standard-view white-bg border absolute standard-view autofill">
         <PreviewItem
-          item={vessels[0]}
+          item={vesselItem}
           itemName={t("NAVIGATION.VESSELS").toUpperCase()}
           icon="vessel"
           previewName="_id"
           subText={t("SEARCH.CATCHES")}
           searchWords={searchWords}
+          onClick={() => goVesselViewPage(vesselItem)}
         />
         <PreviewItem
-          item={crew[0]}
+          item={crewItem}
           itemName={t("SEARCH.CREW_MEMBERS").toUpperCase()}
           icon="crew"
           previewName="name"
           subText={t("NAVIGATION.VESSELS")}
           searchWords={searchWords}
+          onClick={() => goCrewViewPage(crewItem)}
```

Comment:
When I tested this out on the sandbox server, I searched for 'Jack' and got a match:
<img width="550" alt="Screen Shot 2020-10-19 at 10 10 02 PM" src="https://user-images.githubusercontent.com/1767528/96531568-f5c95800-1257-11eb-801a-2ee1ad8a2f6c.png">


I clicked on "Jack Harknass" under "Crew Members" and got this blank page:

<img width="1114" alt="Screen Shot 2020-10-19 at 10 10 09 PM" src="https://user-images.githubusercontent.com/1767528/96531536-dcc0a700-1257-11eb-896c-6b7a1562b069.png">

Note the URL: http://localhost:3000/#/crew/view/{%22captain.name%22:%22Jack%20Harknass%22}

Unfortunately, the crew view page depends on both the crew member's name and the license number of the crew member. Is there any way you can make sure the license number is retrieved on this page, and sent onto the next page?

Reply:
ahh thanks for the catch and the details with screnshots

It should be possible to pass license number (and possibly other details) and have them applied to the filter of the goTo* method

I'll look into it